packages/legend-query/style/_query-builder-functions-explorer.scss

```diff
@@ -0,0 +1,203 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use 'mixins' as *;
+
+.query-builder__functions-explorer__tree {
+  &__root {
+    padding: 0.5rem 0;
+  }
+
+  &__node__icon {
+    width: 4rem;
+    min-width: 4rem;
+  }
+
+  &__expand-icon,
+  &__type-icon {
+    @include flexCenter;
+
+    width: 1.7rem;
+    min-width: 1.7rem;
+    height: 1.4rem;
+  }
+
+  &__node__container:hover {
+    background: var(--color-dark-blue-shade-100);
+  }
+
+  &__node__label--with-action {
+    width: calc(100% - 2rem);
+  }
+
+  &__node__actions {
+    display: flex;
+    height: 2rem;
+    padding-right: 0.5rem;
+  }
+
+  &__node__action {
+    @include flexCenter;
+
+    height: 2rem;
+    width: 2rem;
+
+    svg {
+      font-size: 1.3rem;
+      color: var(--color-light-grey-400);
+    }
+
+    &:hover svg {
+      color: var(--color-light-grey-200);
+    }
+
+    &[disabled] svg {
+      color: var(--color-dark-grey-300);
+    }
+  }
+
+  &__node__info {
+    cursor: help;
+  }
+
+  &__drag-preview-layer {
+    position: fixed;
+    pointer-events: none;
+    z-index: 100;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+  }
+
+  &__drag-preview {
+    @include flexCenter;
+
+    display: inline-flex;
+    height: 2rem;
+    padding: 0 0.5rem;
+    font-size: 1.2rem;
+    background: var(--color-blue-200);
```

Comment:
Let's use `grey` for this. This is no primitive properties.

Reply:
I think we should use blue since this is for dragging preview. It has the same color as the explorer tree does.